### PR TITLE
Fix Sidekiq.options warning for sidekiq-cron v1.6.0

### PR DESCRIPTION
**Pull request by thanhcuong1990**
`Sidekiq.options` warning should be resolved on PR #338. 
But on PR #337 still used `Sidekiq.options` at https://github.com/ondrejbartas/sidekiq-cron/pull/337/files#diff-74c7adba1e1cde8fd26e2a46f0e03b37e9039f3a73ff16776ca21f4555ff7185R6

### CODE REVIEW

**Comment by markets** [comment]
Thanks @thanhcuong1990 for working on this! That's what we said in https://github.com/ondrejbartas/sidekiq-cron/pull/338#issuecomment-1154093167, so I'm +1 to this one! At least we can centralize this logic in one place and easily support different Sidekiq versions.

**Comment by markets** [comment]
@thanhcuong1990 do you think we can also refactor this block?

https://github.com/ondrejbartas/sidekiq-cron/blob/5f6bd48fbcd9cfd041394035294c3e2ecd4beab4/lib/sidekiq/cron/poller.rb#L13-L17

**Comment by thanhcuong1990** [comment]
> @thanhcuong1990 do you think we can also refactor this block?
> 
> https://github.com/ondrejbartas/sidekiq-cron/blob/5f6bd48fbcd9cfd041394035294c3e2ecd4beab4/lib/sidekiq/cron/poller.rb#L13-L17

@markets I've just push a commit to refactor this. Please take a look!

File: lib/sidekiq/options.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'sidekiq'
+
+module Sidekiq
+  module Options
+    def self.[](key)
+      new_version? ? Sidekiq[key] : Sidekiq.options[key]
+    end
+
+    def self.[]=(config, key, value)
+      new_version? ? config[key] = value : config.options[key] = value
```
[the remark this answers is not in the document]
Comment (thanhcuong1990) [reply lib/sidekiq/options.rb]: @markets updated! Thanks for quick feedback. 👍

**Review by markets — APPROVED** [review]

**Comment by markets** [comment]
@danilogco could you please take a look? Thanks!

**Pull request merged**